x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -213,7 +213,7 @@ customise the URIs and their associated value.
 logged in, and they can be used for role mapping (below).
 
 In order for these attributes to be useful, {es} and the IdP need to have a
-common via for the names of the attributes. This is done manually, by
+common value for the names of the attributes. This is done manually, by
```

Comment:
I have no idea what I was trying to write here, but `value` seems like a good enough guess.

Reply:
> I have no idea what I was trying to write here

`common view` maybe ?

Reply:
Maybe. I considered that, but I think I would have written `view of` rather than `view for`.
🤷‍♂️